language-server/language-server-api/src/main/scala/aqua/lsp/LspContext.scala

```diff
@@ -22,7 +22,8 @@ case class LspContext[S[_]](
   variables: List[VariableInfo[S]] = Nil,
   importTokens: List[LiteralToken[S]] = Nil,
   errors: List[SemanticError[S]] = Nil,
-  warnings: List[SemanticWarning[S]] = Nil
+  warnings: List[SemanticWarning[S]] = Nil,
+  importPaths: Map[String, String] = Map.empty
```

Comment:
Why not merge it together with `importTokens`?

Reply:
they are gathered from different sources

Reply:
BTW, what will happen, if two files are compiled as dependencies, both have `import "some.aqua"`, but imports are resolved to different paths? `importPaths` map is just concatenated when merging two contexts. However, this should not affect anything as this info is used only for the file being viewed? :thinking:

Reply:
yes, you are right

Reply:
> So? I think it is much more clear to do something like
> 
> ```scala
> case class Import[S[_]](
>   token: LiteralToken[S],
>   resolution: Option[String] = None
> ) {
>   def resolve(resolution: String): Import[S] = copy(resolution = resolution.some)
> }
> ```

i don't like it. I should work with options two times (in setImportPaths and in converting imports to JS) instead of merge two fields at the end (ResultHelper) once.